### ISSUE TITLE
Mac OS X Catalina breaks default self signed certificates

### ISSUE THREAD

**Issue by patrick0057** [rancher/rancher#23584]
**What kind of request is this (question/bug/enhancement/feature request):**
Bug

**Steps to reproduce (least amount of steps as possible):**
1) Upgrade Mac OS X to 10.15 Catalina
2) Install Rancher HA cluster using rancher generated self signed certificates
**Result:**
You will be unable to get past the security warning in Google Chrome without installing the certificate locally.  You will see an error as shown below.
```
Your connection is not private
Attackers might be trying to steal your information from myrancher.com (for example, passwords, messages, or credit cards). Learn more
NET::ERR_CERT_INVALID

Help improve Safe Browsing by sending some system information and page content to Google. Privacy policy
myrancher.com normally uses encryption to protect your information. When Google Chrome tried to connect to myrancher.com this time, the website sent back unusual and incorrect credentials. This may happen when an attacker is trying to pretend to be myrancher.com, or a Wi-Fi sign-in screen has interrupted the connection. Your information is still secure because Google Chrome stopped the connection before any data was exchanged.

You cannot visit myrancher.com right now because the website sent scrambled credentials that Google Chrome cannot process. Network errors and attacks are usually temporary, so this page will probably work later.
```

**Other details that may be helpful:**
Apple has added new security requirements in Catalina which can be found in the following URL.
https://support.apple.com/en-us/HT210176

**Environment information**
- Rancher version (`rancher/rancher`/`rancher/server` image tag or shown bottom left in the UI): v2.3.0
- Installation option (single install/HA): HA

**Comment by Oats87** [comment]
I believe part of the issue is missing the Extended Key Usage for Server Auth. I looked into trying to have cert-manager sign a cert with it but it looks to be less-than-straightforward.

**Comment by munnerz** [comment]
This is fixed as of cert-manager v0.11.1 and v0.12.0 😄 see https://github.com/jetstack/cert-manager/pull/2351

**Comment by john-terrell** [comment]
BTW, still seeing this on v0.15.0 of cert-manager.

**Comment by MTijbout** [comment]
> BTW, still seeing this on v0.15.0 of cert-manager.

Indeed. I face the same issue with Chrome. With FireFox I can "continue to unsafe site"
